DanielVargas/space_challenge/src/com/company/Rocket.java

```diff
@@ -0,0 +1,71 @@
+package com.company;
+/**
+ * Created by Daniel Vargas on 12/05/2020.
+ */
+public class Rocket implements SpaceShip {
```

Comment:
Please document your code. You can find a guideline in this [link](https://www.spiria.com/en/blog/method-and-best-practices/how-document-java-code/)